README.md

```diff
@@ -47,58 +45,27 @@ When building on Linux it is sometimes necessary to run:
     $ gradle wrapper
     
 Finally, turn on the "Use the new formplayer frontend" feature flag on your CommCareHQ domain
+
+### Contributing
+
+For PRs that just modify code in the Formplayer repo, submit a PR to Formplayer on a separate branch.
+
+
+#### Contributing changes to commcare
+
+Formplayer also has a dependency on the commcare-core repository. The commcare-core `master` branch is not
+stable and Formplayer uses a different branch. The submodule repo `libs/commcare` should always be pointing to
+the `formplayer` branch.
```

Comment:
i do agree it'd be nice to keep formplayer on the commcare release schedule. maybe this will be easier once it's stabilized a bit. it's a little harder since web really doesn't have any restrictions when releasing things, so whenever we get a ticket, we can either branch off from commcare release cycle and fix it right away or wait a month for the next mobile release. i think we'd just have to set those expectations if that's the route we want to go long term